src/Elcodi/Component/Core/Tests/UnitTest/Entity/Traits/IdentifiableTrait.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Elcodi package.
+ *
+ * Copyright (c) 2014-2015 Elcodi Networks S.L.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Feel free to edit as you please, and have fun.
+ *
+ * @author Marc Morera <yuhu@mmoreram.com>
+ * @author Aldo Chiecchia <zimage@tiscali.it>
+ * @author Elcodi Team <tech@elcodi.com>
+ */
+
+namespace Elcodi\Component\Core\Tests\UnitTest\Entity\Traits;
+
+trait IdentifiableTrait
```

Comment:
Should I keep the `EnabledTrait`? There are also the `enable` and `disable` tests there.